.gitignore

```diff
@@ -104,3 +104,4 @@ kuma/static/js/libs/ckeditor/source/ckbuilder/
 
 # Emacs backup files
 *~
+junit.xml
```

Comment:
This change is for when you run `jest` and it fails to execute. For example, on my macOS I now have Node 13 which has compatibility problems with the `node-sass` package. So every time I try to run `jest -b` on the host it leaves this file.